core/src/main/kotlin/pages/ContentNodes.kt

```diff
@@ -318,7 +318,7 @@ interface Kind
 
 enum class ContentKind : Kind {
 
-    Comment, Constructors, Functions, Parameters, Properties, Classlikes, Packages, Symbol, Sample, Main, BriefComment,
+    Comment, Constructors, Functions, Parameters, Parameter, Properties, Classlikes, Packages, Symbol, Sample, Main, BriefComment,
```

Comment:
It is really confusing with `Parameters` that has another application.
You can create another `Kind` or style.

Reply:
Discussed in person. `ContentKind` seems to be about big blocks of content. For instance, `ContentKind.Parameter` is this:
![parameterscontent](https://user-images.githubusercontent.com/22685910/149769021-571bb2e7-f81d-4c20-9a32-3127d0096463.png)

I thought `ContentKind` was more of a universal way to mark any content anywhere, so it is indeed strange to use it for function parameters. Will re-do to `SymbolStyle` and write KDocs

Reply:
Introduced `SymbolContentKind`, added KDocs to avoid such misunderstandings in the future